### PR TITLE
Update openldap.md

### PR DESCRIPTION
**Pull request by bendeg**
I would like to add a little explanation for "memberOf" in bootstrap.ldif step because, for a non-expert person, it doesn't tell very clearly how to find out the name we have to use.

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Thanks !

**Comment by bendeg** [comment]
I closed this PR before it was merged : re-opening it, sorry...

**Comment by fvanderbiest** [comment]
@pmauduit can you check this is correct ? Thanks.

**Comment by landryb** [comment]
This applies mostly only to the default debian setup (of creating a database upon install), but since it's more or less what we target..

**Comment by bendeg** [comment]
I don't mind adding a comment that specifically targets debian users  if needed. Just ask and I'll modify accordingly

File: docs/setup/openldap.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ sudo ldapadd -Y EXTERNAL -H ldapi:/// -f bootstrap.ldif
 
 If successful, the above command should display: ```adding new entry "olcDatabase=mdb,cn=config"```.
 
+Now, be careful : after a fresh slapd install there's already a olcDatabase={1}mdb.ldif
```
Comment (pmauduit) [inline docs/setup/openldap.md]: I would say instead:

"After a fresh slapd install, you might already have a database set up for you, depending on how the package manager of the distribution is configured, or if the package planned to do so.

To check the database number in your LDAP setup, you can go through the `/etc/ldap/slapd.d/cn=config/` subdirectory and check the files `olcDatabase={x}mdb.ldif`". The different files should contain a `olcSuffix` attribute. The expected geOrchestra database is the one which contains a `olcSuffix: dc=georchestra,dc=org`.

the memberOf.ldif file should match the expected database number in OpenLDAP, and you might need to adapt this LDIF file.
"

Comment (bendeg) [reply docs/setup/openldap.md]: +1
Indeed, that's THE answer : checking the file that contains right olcSuffix.

I may seem obvious for those who knows ldap, it is not for others, like me...

**Pull request closed without merging**